RecoVertex/BeamSpotProducer/plugins/OfflineToTransientBeamSpotESProducer.cc

```diff
@@ -0,0 +1,37 @@
+#include "RecoVertex/BeamSpotProducer/plugins/OfflineToTransientBeamSpotESProducer.h"
```

Comment:
This could be just
```suggestion
#include "OfflineToTransientBeamSpotESProducer.h"
```
or, even better, the contents of the header included in this file (as recommended in the code rules).